File: app/src/main/java/com/ronju/covid_19tracker/Activitys/Fragment/HealthCareActivity.java
package com.ronju.covid_19tracker.Activitys.Fragment;

import androidx.browser.customtabs.CustomTabsIntent;
import androidx.fragment.app.Fragment;

import android.content.Intent;
import android.net.Uri;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;
import android.widget.ImageView;
import android.widget.TextView;

import com.ronju.covid_19tracker.R;

public class HealthCareActivity extends Fragment {
    ImageView who1;
    TextView viewAll;
    private final String HOTLINES_NUMBER = "https://corona.gov.bd/call_center";
    private final String TEST_BY_OWN ="https://corona.gov.bd/corona-symptom";
    private final String APPLY_VACCINE ="https://surokkha.gov.bd/enroll";
    private final String BRAC_TEST_APPLY="https://coronatest.brac.net";
    Button callNow,hotLines,ownTestBtn,vaccineBtn,testBtn;

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.activity_health_care, container, false);

        initView(view);


        callNow.setOnClickListener(v->{
            Intent intent = new Intent(Intent.ACTION_DIAL);
            intent.setData(Uri.parse("tel:33313"));
            startActivity(intent);

        });
        hotLines.setOnClickListener(v->{
            openBrowserTab();
        });

        ownTestBtn.setOnClickListener(v->{
            CustomTabsIntent.Builder builder = new CustomTabsIntent.Builder();
            CustomTabsIntent customTabsIntent = builder.build();
            customTabsIntent.launchUrl(getActivity(),Uri.parse(TEST_BY_OWN));
        });

        vaccineBtn.setOnClickListener(v->{
            CustomTabsIntent.Builder builder = new CustomTabsIntent.Builder();
            CustomTabsIntent customTabsIntent = builder.build();
            customTabsIntent.launchUrl(getActivity(),Uri.parse(APPLY_VACCINE));
        });

        testBtn.setOnClickListener(v->{
            CustomTabsIntent.Builder builder = new CustomTabsIntent.Builder();
            CustomTabsIntent customTabsIntent = builder.build();
            customTabsIntent.launchUrl(getActivity(),Uri.parse(BRAC_TEST_APPLY));
        });

        return view;
    }

    private void openBrowserTab() {
        CustomTabsIntent.Builder builder = new CustomTabsIntent.Builder();

        CustomTabsIntent customTabsIntent = builder.build();
        customTabsIntent.launchUrl(getContext(), Uri.parse(HOTLINES_NUMBER));
    }

    private void initView(View view) {
        callNow = view.findViewById(R.id.callNow);
        hotLines = view.findViewById(R.id.hotLines);
        ownTestBtn = view.findViewById(R.id.ownTestBtn);
        vaccineBtn = view.findViewById(R.id.applyVaccine);
        testBtn = view.findViewById(R.id.applyForTest);
    }
}

File: app/src/main/java/com/corona/covid_19tracker/Activity/Fragment/HealthCareActivity.kt
package com.corona.covid_19tracker.Activity.Fragment

import android.widget.TextView
import android.view.LayoutInflater
import android.view.ViewGroup
import android.os.Bundle
import com.corona.covid_19tracker.R
import android.content.Intent
import android.net.Uri
import android.view.View
import android.widget.Button
import android.widget.ImageView
import androidx.browser.customtabs.CustomTabsIntent
import androidx.fragment.app.Fragment
import com.corona.covid_19tracker.Activity.WebViewActivity
import com.corona.covid_19tracker.Units.Unit
import com.corona.covid_19tracker.databinding.ActivityHealthCareBinding

class HealthCareActivity : Fragment() {
    private lateinit var binding: ActivityHealthCareBinding
    private lateinit var intent: Intent
    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        binding = ActivityHealthCareBinding.inflate(inflater, container, false)
        intent = Intent(context, WebViewActivity::class.java)
        binding.callNow.setOnClickListener {
            val intent = Intent(Intent.ACTION_DIAL)
            intent.data = Uri.parse("tel:33313")
            startActivity(intent)
        }
        binding.hotLines.setOnClickListener {
            intent.putExtra("url", Unit.HOTLINES_NUMBER)
            startActivity(intent)
        }
        binding.ownTestBtn.setOnClickListener {
            intent.putExtra("url", Unit.TEST_BY_OWN)
            startActivity(intent)
        }
        binding.applyVaccine.setOnClickListener {
            intent.putExtra("url", Unit.APPLY_VACCINE)
            startActivity(intent)

        }
        binding.applyForTest.setOnClickListener {
            intent.putExtra("url", Unit.BRAC_TEST_APPLY)
            startActivity(intent)

        }
        return binding.root
    }
}
